pylint_nautobot/constants.py

```diff
@@ -0,0 +1,144 @@
+"""Define constants for the pylint-nautobot plugin."""
+
+# TODO Remove after dropping support for Python 3.8
+from __future__ import annotations
+
+from pylint.typing import MessageDefinitionTuple
+
+
+class MESSAGES:  # pylint: disable=too-few-public-methods
```

Comment:
Is there prior art for this in pylint or other pylint projects? I think we should try to be consistent with the parent library and their documentation in cases like pylint where the learning curve is already a cliff.

Reply:
Similar in their examples and their codebase. The difference is that they use multiple codes in a single class attribute, and they inherit from BaseChecker, but also combine some utils. I will post their code. I decided on using a separate attribute for each code to help solve the issue of finding all the codes used.

Here is an example, perhaps I could change this to MESSAGESMixin:

https://github.com/pylint-dev/pylint/blob/main/pylint/checkers/deprecated.py

Reply:
We decided after discussion with @cmsirbu and @glennmatthews that @smk4664 will have to do the work to revert these changes and we'll get a follow-up issue opened to introduce an invoke command to list out the current pylint message codes (`pylint --list-msgs-enabled | grep 'nb-'`)